src/resources/js/components/Presentation.vue

```diff
@@ -93,6 +99,24 @@ export default {
           alert.showMessage('error', 'Oops une erreur c\'est produite lors de la désinscription');
         });
     },
+    isPresenter() {
+      return (this.presentation.auth_user_role === 'presenter');
+    },
+    delPresentation(id) {
+      const apiUrl = `/presentations/${id}`;
+
+      window.axios.delete(apiUrl)
+        .then(() => {
+          // go back to presentation list
+          alert.showMessage('success', 'La presentation à été supprimé correctement');
+          setTimeout(() => {
+            this.$router.push('/presentations');
+          }, 2000);
+        })
+        .catch((error) => {
+          alert.showMessage('error', `echec de la suppression : ${error.status}`);
```

Comment:
do not show backend error to the final user, errors from backend are used to help frontend developpers to code their interfaces, not to interract with the application users

Reply:
corrected in [ce4d0](https://github.com/kayoumido/HEIGVD-PRO-A-11/pull/161/commits/ce4d040aa05f18cb444f29d0d1b1d1e7958da556)